Tasks/HelmDeployV0/src/helm.ts

```diff
@@ -165,7 +165,8 @@ function runHelm(helmCli: helmcli, command: string, kubectlCli: kubernetescli, f
     else if ((command === "install" || command === "upgrade")) {
```

Comment:
Why are there double brackets?

Reply:
They shouldn't be there. I'll remove them.